lib/evaluate.js

```diff
@@ -239,6 +240,19 @@ export const runFraudDetection = async (roundIndex, measurements, sparkRoundDeta
     }
   }
 
+  //
+  // 3. Reward only first TN measurements
+  //
+  const tasksPerNode = new Map()
+  for (const m of measurements) {
+    if (m.fraudAssessment && m.fraudAssessment !== 'OK') continue
+    const node = `${m.inet_group}${m.participantAddress}`
```

Comment:
I saw that, but didn't bother here because those are never printed. I'll add them for consistency 👍

Reply:
FWIW, I am adding these delimiters to prevent a (very unlikely) edge case where we have two different combinations of `(inet_group, participantAddress)` strings that produce the same string when joined together. Rather than reason in every place about the probability of such collision, I add the delimiter as a universal defensive move.